### DISCUSSION TITLE
OR creation difficulties

### DISCUSSION THREAD

**Discussion by AcooLsnEaKerS** [SebLague/Digital-Logic-Sim#208] [Q&A]
**EDIT:** Went away from the PC for awhile and came back - I think I got it.
Technically no output is still an output so that's why the light blinks between the two states, since it gets confused.

Hi!
I just discovered the app and wanted to give it a shot myself. I think it is fair to say from the screenshot that I don't have much experience with things like these but I'd like to learn. I wanted to create an OR statement which I can use later in other simulations but the output just blinks whenever I turn the A or/and B on, instead of staying in on state. 
I saw how people have made their own ORs with NANDs (after making the NAND) but if I need to be honest - I can't fully wrap my head around the solutions.
This is my attempt for the OR statement/module/chip/etc.
My logic behind it was that I had to cover the truth table of the or so
- if A is on and B is on -> output should be on
- if A is off but B is on -> output should be on
- if A is on but B is off -> output should be on
- if A is off and B is off -> output should be off

I'd appreciate any help I could get - be it an explanation on why my attempt doesn't function as I expected or an explanation on the NAND way of solving it (link to the discussion I am referring https://github.com/SebLague/Digital-Logic-Sim/discussions/54)
I get how the NAND works - it's basically a reversed AND but then I get lost when it comes to the making of the OR.

![image](https://github.com/SebLague/Digital-Logic-Sim/assets/158824711/02338642-9b93-494a-82ee-e27fe2763166)
![image2](https://github.com/SebLague/Digital-Logic-Sim/assets/158824711/3a3fa1e6-3d17-4f0e-9bed-19df49747200)

**Comment by MF9744** [comment]
explore Tristate buffer
